### DISCUSSION TITLE
Download model in base model subfolders

### DISCUSSION THREAD

**Discussion by csprsmt** [Comfy-Org/ComfyUI-Manager#1081] [Ideas]
Models downloaded with the model manager often don't display the base model they are compatible with in the name. It would be convenient if they are automatically placed within a subfolder of the given base model, like sdxl or flux. This makes it easier to select compatible models and reduces the lists. Yes, I could rename models and/or place them in subfolders myself. But then the model manager can't see I already downloaded them.

**Comment by ltdrdata** [comment]
The model management function is planned to be removed from ComfyUI-Manager in the future. This functionality will ultimately be managed by ComfyUI core.

https://github.com/Comfy-Org/ComfyUI_frontend/pull/569
